_webi/template.sh

```diff
@@ -448,11 +445,10 @@ __bootstrap_webi() {
         webi_extract
     }
 
-    # move commands from the extracted archive directory to $HOME/.local/opt or $HOME/.local/bin
-    # shellcheck disable=2120
-    # webi_install may be sourced and used elsewhere
+    # move commands from the extracted archive directory
+    # to $HOME/.local/opt or $HOME/.local/bin
     webi_install() {
-        if [ -n "$WEBI_SINGLE" ] || [ "single" = "${1-}" ]; then
+        if test -n "${WEBI_SINGLE}"; then
```

Comment:
verified that `webi_install` isn't used externally and removed shellcheck comment